content/docs/started/getting-started-gke.md

```diff
@@ -33,15 +33,18 @@ Create an OAuth client ID to be used to identify Cloud IAP when requesting acces
    * Configure the [consent screen](https://console.cloud.google.com/apis/credentials/consent).
    * Under **Email address**, select the address that you want to display as a public contact. You must use either your email address or a Google Group that you own.
    * In the **Product name** box, enter a suitable name like `kubeflow`.
-   * Under **Authorized domains**, enter
+   * If you see **Authorized domains**, enter
 
      ```
      <project>.cloud.goog
      ```
-
-     where \<project\> is your Google Cloud Platform (GCP) project ID.
-
+     * where \<project\> is your Google Cloud Platform (GCP) project ID.
+     * If you are using your own domain e.g. **acme.com** you should add that as well
+     * Depending on your project **Authorized domains** might not be an option
    * Click Save.
+   * Here's a screenshot
+   
+     ![consent-screen](/docs/images/consent-screen.png)
```

Comment:
The screenshot shows an authorized domain = "kubeflow.org" but our instructions say it must follow the pattern "<project>.cloud.goog". If "kubeflow.org" is an example of "using your own domain", then we should use something other than "kubeflow.org" as the example. It's confusing as it is, because people may justifiably think they need to enter "kubeflow.org".

Reply:
Good point.